README.md

```diff
@@ -1,9 +1,18 @@
 # sw-testing-helpers
 
-A set of helper files used to test Propel and sw-toolbox.
+[![Build Status](https://travis-ci.org/GoogleChrome/sw-testing-helpers.svg?branch=master)](https://travis-ci.org/GoogleChrome/sw-testing-helpers) [![Dependency Status](https://david-dm.org/GoogleChrome/sw-testing-helpers.svg)](https://david-dm.org/GoogleChrome/sw-testing-helpers) [![devDependency Status](https://david-dm.org/GoogleChrome/sw-testing-helpers/dev-status.svg)](https://david-dm.org/GoogleChrome/sw-testing-helpers#info=devDependencies)
+
+This Repo is largely to share code across several libraries being developed
+by the Google Chrome DevRel team.
```

Comment:
Maybe "Web DevRel" instead of "Chrome DevRel"? To emphasize that we care about the whole web, not specific Chrome.